packages/runtime/container-runtime/src/summary/runningSummarizer.ts

```diff
@@ -817,6 +829,11 @@ export class RunningSummarizer extends TypedEventEmitter<ISummarizerEvents> impl
 
 		// If summarization is still unsuccessful, stop the summarizer.
 		if (status === "failure") {
+			this.mc.logger.sendErrorEvent({
```

Comment:
Should we also log success here for the oveall summary? Does the current `Summarize_end` event indicate that today?